source.txt

```diff
@@ -204,7 +204,8 @@ Table of Contents
 
     All folders are treated as existing, and therefore GET requests to
     untouched folders SHOULD be responded to with a folder description
-    with no items (the items field set to '{}'). However, an empty
+    with no items (the items field set to '{}'). ETags of empty folders
+    MUST NOT change value for successive GET requests. However, an empty
     folder MUST NOT be listed as an item in its parent folder.
```

Comment:
>That would mean that if a file is created and then deleted in a previously empty folder, subsequent GET requests would get a different ETag.

This is actually already allowed with the current PR, because *inbetween*, the folder wasn't empty. The important thing here is that the etag doesn't change if I request a empty folder with GET multiple times, with no other requests inbetween.